README.md

```diff
@@ -32,84 +48,186 @@ spec:
         "bridge": "br1"
       },
       {
-        "type": "cidr-filtering-cni"
+        "type": "bridge-filtering"
       }
     ]
   }'
+```
+
+The `example-network` specifies two CNI plugins, the first cni in the above is a `cnv-bridge`, connecting pods to linux-bridge `br1`, and the second
+is the `bridge-filtering`, that handles configuration of nftables rules on the pod.
+
+By default, all ingress and egress traffic is dropped.
```

Comment:
I think it' not 100% correct, as it makes it sound that the default is applied only one a configMap configuration references the NAD, but adding the plugin in the NAD is enough

Reply:
I'm just trying to avoid the reader misunderstanding that once this plugin is installed "all traffic is dropped" by default.

What if we remove the "one of" from my suggestion ?